src/components/link-preview/link-preview.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import LinkPreview from ".";
+
+export const DefaultStory: ComponentStory<typeof LinkPreview> = () => {
+  return (
+    <LinkPreview
+      title="This is an example of a title"
+      url="https://www.sage.com"
+      description="Captain, why are we out here chasing comets?"
+    />
+  );
+};
+
+export const LinkPreviewLoadingState: ComponentStory<
+  typeof LinkPreview
+> = () => {
+  return (
+    <LinkPreview
+      title="This is an example of a title"
+      url="https://www.sage.com"
+      description="Captain, why are we out here chasing comets?"
+      isLoading
```

Comment:
Do you need the title, url and description if the Link Preview is in loading state?